src/Utils/StringUtils.php

```diff
@@ -28,4 +28,41 @@ public static function csvToArray($args)
         //
         return array_map('trim', array_filter(explode(',', is_array($args) ? implode(',', $args) : $args)));
     }
+
+    /**
+     * Replace placeholders in a string.
```

Comment:
Approved. Ideally we add @param annotations to the new method. (I feel odd asking the annotation maintainer about annotations)